docs/http/traffic-policy/actions/custom-response.mdx

```diff
@@ -24,8 +24,9 @@ If this action is executed, no further actions in the traffic policy will be exe
 | ----------------- |
 | `custom-response` |
 
-| Parameter      | &nbsp; | Description                                                                                                        |
-| -------------- | ------ | ------------------------------------------------------------------------------------------------------------------ |
-| `status_code`  | int    | The status code of the response.                                                                                   |
-| `content_type` | string | The content type of the response body, which will be set on the Content-Type header. Required if `content` is set. |
-| `content`      | string | The body of the response.                                                                                          |
+| Parameter      | &nbsp;                    | Description                                                                                                                                                          |
+| -------------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `status_code`  | int                       | The status code of the response.                                                                                                                   |
+| `content`      | string                    | The body of the response.                                                                                                                          |
+| `headers`      | Map&lt;string, string&gt; | Headers to be added to the response. If `content-type` is not included in `headers`, this action will attempt to infer the correct `content-type`. |
```

Comment:
@nijikokun do we want to explicitly call out that this will infer content-type here?

Reply:
i was thinking it'd be good to document somewhere. maybe this isn't the place. also maybe we don't need to document it at all, maybe its absence is enough to imply to the user that they don't need it unless they really care

Reply:
Yes, this is good to know for our end users